controllers/TuitController.ts

```diff
@@ -132,4 +158,17 @@ export default class TuitController implements ITuitController {
     TuitController.tuitDao
       .deleteTuit(req.params.tid)
       .then((status) => res.send(status));
+
+  /**
+   * @param {Request} req Represents request from client, including path
+   * parameter tid identifying the primary key of the tuit whose 
+   * version history to be viewed.
+   * @param {Response} res Represents response to client, array of all the versions of the 
+   * Tuit.
+   */
+  getVersions = (req: Request, res: Response) =>{
+    TuitController.tuitVersionDao.findAllPreviousVersions(req.params.tid)
```

Comment:
`TuitVersionDao` will only fetch previous versions of the tuit. Here, we also want to include the current version of tuit. So the response will be generated by appending the outputs from TuitDao's findTuitById and TuitVersionDao's findPreviousVersions methods